contracts/JBFundingCycleStore.sol

```diff
@@ -97,22 +99,37 @@ contract JBFundingCycleStore is JBControllerUtility, IJBFundingCycleStore {
 
     @return _fundingCycle The queued funding cycle.
   */
-  function queuedOf(uint256 _projectId) public view override returns (JBFundingCycle memory) {
+  function queuedOf(uint256 _projectId)
+    external
+    view
+    override
+    returns (JBFundingCycle memory _fundingCycle)
+  {
     // The project must have funding cycles.
     if (latestConfigurationOf[_projectId] == 0) return _getStructFor(0, 0);
 
     // Get a reference to the configuration of the standby funding cycle.
     uint256 _fundingCycleConfiguration = _standbyOf(_projectId);
```

Comment:
nit) I feel the readability of this fn suffers by continuously updating this variable. Personally I would prefer to declare it as needed with a meaningful name that represents the context, or just inline whatever it's set to when it's used.

for example:

_fundingCycleConfiguration = _fundingCycle.basedOn;
// Resolve the funding cycle for the for the latest configured funding cycle.
_fundingCycle = _getStructFor(_projectId, _fundingCycleConfiguration);

becomes:

// Resolve the funding cycle for the for the latest configured funding cycle.
_fundingCycle = _getStructFor(_projectId, _fundingCycle.basedOn);

Reply:
great call